### PR TITLE
Migrate codec-http tests to JUnit 5

### PR DESCRIPTION
**Pull request by kashike**
https://github.com/netty/netty/issues/10757#issuecomment-848445551

### CODE REVIEW

**Review by normanmaurer — APPROVED** [review]

**Comment by kashike** [comment]
Oops, I missed those two cases - sorry!

File: codec-http/src/test/java/io/netty/handler/codec/http/CombinedHttpHeadersTest.java [hunk 1]
```diff
@@ -140,11 +141,11 @@ public void addCharSequencesCsvWithValueContainingCommas() {
         assertEquals(HeaderValue.EIGHT.subset(6), headers.getAll(HEADER_NAME));
     }
 
-    @Test (expected = NullPointerException.class)
+    @Test
     public void addCharSequencesCsvNullValue() {
         final CombinedHttpHeaders headers = newCombinedHttpHeaders();
         final String value = null;
-        headers.add(HEADER_NAME, value);
+        assertThrows(NullPointerException.class, () ->headers.add(HEADER_NAME, value));
```
Comment (hyperxpro) [inline codec-http/src/test/java/io/netty/handler/codec/http/CombinedHttpHeadersTest.java:148]: Add space after `->`.

Comment (kashike) [reply codec-http/src/test/java/io/netty/handler/codec/http/CombinedHttpHeadersTest.java]: Resolved.

**Review by hyperxpro — APPROVED** [review]

**Review by normanmaurer — APPROVED** [review]

**Comment by normanmaurer** [comment]
@kashike thanks a lot!

**Pull request merged**